observatorium/overlays/moc/smaug/loki/configmaps/observatorium-loki-configmap_patch.yaml

```diff
@@ -120,12 +121,7 @@ data:
           "consistent_hash": true
     "tracing":
       "enabled": false
-  overrides.yaml: '{}'
 kind: ConfigMap
 metadata:
-  labels:
-    app.kubernetes.io/instance: observatorium
-    app.kubernetes.io/name: loki
-    app.kubernetes.io/part-of: observatorium
-    app.kubernetes.io/version: 2.3.0
   name: observatorium-loki
+  namespace: opf-observatorium
```

Comment:
drop the namespace here as well.

Reply:
Leaving this unresolved for the same issue as above, but I think the above issue should be solved by the current state of the PR.

Reply:
Yeah getting the same error as before... here are the logs of the `kustomize-build` check, reverting this change for now: ```
accumulating resources: accumulation err='accumulating resources from 'loki': '/home/prow/go/src/github.com/operate-first/apps/observatorium/overlays/moc/smaug/loki' must resolve to a file': recursed accumulation of path '/home/prow/go/src/github.com/operate-first/apps/observatorium/overlays/moc/smaug/loki': no matches for IdId ~G_v1_ConfigMap|opf-observatorium|observatorium-loki; failed to find unique target for patch ~G_v1_ConfigMap|observatorium-loki
Aborting due to errexit on line [4](https://prow.operate-first.cloud/view/s3/ci-prow/prow-logs/pr-logs/pull/operate-first_apps/2253/kustomize-build/1564763177680375808#1:build-log.txt%3A4)3. Exit code: 1
```